src/tools/wasm-metadce.cpp

```diff
@@ -578,6 +605,10 @@ int main(int argc, const char* argv[]) {
     ModuleWriter writer;
     writer.setBinary(emitBinary);
     writer.setDebugInfo(debugInfo);
+    if (outputSourceMapFilename.size()) {
+      writer.setSourceMapFilename(outputSourceMapFilename);
+      writer.setSourceMapUrl(outputSourceMapUrl);
+    }
```

Comment:
Please add a test for `wasm-metadce` too.

Reply:
I have also updated the test for `wasm-metadce` to test the `-ism` and `-osm` flags as well.